app/src/main/java/org/gdg/frisbee/android/fragment/PulseFragment.java

```diff
@@ -139,16 +151,15 @@ public void onNotFound(String key) {
         });
     }
 
-    private void initAdapter(Pulse pulse) {
-        mPulse = pulse;

```

Comment:
Oh I see :)

On Thu, Feb 26, 2015, 7:16 PM Henrique Rocha <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/fragment/PulseFragment.java
> <https://github.com/gdg-x/frisbee/pull/230#discussion_r25445025>:
>
> > @@ -139,16 +151,15 @@ public void onNotFound(String key) {
> >          });
> >      }
> >
> > -    private void initAdapter(Pulse pulse) {
> > -        mPulse = pulse;
>
> Because mPulse wasn't used. It could be converted to a local variable,
> which wasn't actually used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/230/files#r25445025>.
>